04-Intro_to_Vue/assignment/html-vue/src/components/products.vue

```diff
@@ -0,0 +1,30 @@
+<script setup>
+import { ref } from "vue";
+import img1 from "./images/develop.jpg";
+import img2 from "./images/analyse.jpg";
+import img3 from "./images/develop.jpg";
+const name = ref ("products");
+const images = ref(['img1', 'img2', 'img3']);
```

Comment:
`const images = [
    { src: "./images/develop.jpg",
      alt: "A photo of develop"
    },
    { 
    }
]`

`  <li v-for='image in images' :key='image'>
              <img :src='image.src' alt='image.alt' />
            </li>`